src/leappto/providers/libvirt_provider.py

```diff
@@ -98,7 +98,9 @@ def __inspect_os(domain_name):
             :param domain_name: str, which domain to inspect
             :return:
             """
-            cmd = ['virt-inspector', '-d', domain_name]
+            cmd = ['virt-inspector', '-d', domain_name, '--no-icon']
```

Comment:
yes

    $ virt-inspector --help
    virt-inspector: display information about a virtual machine
    Copyright (C) 2010 Red Hat Inc.
    Usage:
      virt-inspector [--options] -d domname file [file ...]
      virt-inspector [--options] -a disk.img [-a disk.img ...] file [file ...]
    Options:
      -a|--add image       Add image
      -c|--connect uri     Specify libvirt URI for -d option
      -d|--domain guest    Add disks from libvirt guest
      --echo-keys          Don't turn off echo for passphrases
      --format[=raw|..]    Force disk format for -a option
      --help               Display brief help
      --keys-from-stdin    Read passphrases from stdin
      --no-applications    Do not output the installed applications
      --no-icon            Do not output the guest icon
      -v|--verbose         Verbose messages
      -V|--version         Display version and exit
      -x                   Trace libguestfs API calls
      --xpath query        Perform an XPath query
    For more information, see the manpage virt-inspector(1).